jina/types/querylang/__init__.py

```diff
@@ -96,10 +99,19 @@ def priority(self, value: int):
 
     @property
     def name(self) -> str:
-        """Get the name of the driver that the query language attached to """
+        """
+        Get the name of the driver that the query language attached to
+        
+        :rtype: str
```

Comment:
it's a bit wired to see rtype without return

Reply:
I was going to add it, but thanks for the reminder